tests/v2_tests/test_create_index.py

```diff
@@ -6,9 +6,10 @@
 from tests.marqo_test import MarqoTestCase
 
 
+@mark.fixed
```

Comment:
feel free to remove the mark if you believe all the tests are fixed. Also update mark logic in the conftest.py

Reply:
Not all of them are resolved as some of them are bugs on marqo side like number of deleted docs is incorrect in response so I keep tag for now, we can remove it after fixing some of remaining tests in next marqo release that would fix some of the bugs that needs to be addressed.